Add an initial README.md file

jschwartzenberg opened the pull request:
@stsp let me know what you think :)

I'll update the commit if there's anything missing.

stsp commented:
I added license now myself
so that it go with the separate
commit. So please update
the commit msg.

stsp commented:
I think you should disable this:
```
setupapps                  ; Setup applications already on hard disk
autosetupapps              ; Set up all applications on hard disk
```
IIRC it scans the drives for
the known apps. We don't
need that, and who knows
what happens when some
host drives are scanned and
how long will that take.

stsp commented:
`;endopt      = restart`
Why this is commented?
Or is it "exit" by default?

jschwartzenberg commented:
> I added license now myself so that it go with the separate commit. So please update the commit msg.

Great! I just rebased the branch, updated the message and the PR title. Let me know if there's anything else I should alter!

stsp commented:
Thanks!
Looking forward for more work on this. :)

jschwartzenberg commented:
> I think you should disable this:
> 
> ```
> setupapps                  ; Setup applications already on hard disk
> autosetupapps              ; Set up all applications on hard disk
> ```
> 
> IIRC it scans the drives for the known apps. We don't need that, and who knows what happens when some host drives are scanned and how long will that take.

That's true. When you run this after `insfdusr.bat` it will also find some applications and ask to set up shortcuts for them:
![afbeelding](https://github.com/user-attachments/assets/220a512b-4cba-493b-84b9-e238a6da17c0)

It recognizes `sc` as a Scheduler (seems wrong) and the FreeDOS editor as IBM's DOS editor. And `ping` is recognized as FTP PING. I can create a PR to comment them out.

jschwartzenberg commented:
> `;endopt = restart` Why this is commented? Or is it "exit" by default?

I don't remember now. Right now it asks the user. Which option do you think would be the most suitable? Note that the PATH is appended also after the installation completes, so for that a restart wouldn't be necessary. Is there any reason to restart instead of exit?

stsp commented:
> I can create a PR to comment them out.

If its not possible to specify the
fixed set of dirs, then yes. Scanning
host drives (that can be multi-giga
and have backward symlinks) is an
absolute no-no.

> PATH is appended also after the installation completes, so for that a restart wouldn't be necessary.

I think you confuse "restart" with
"reboot". I have no idea what does
"restart" have with PATH.

> Is there any reason to restart instead of exit?

Nope.

jschwartzenberg commented:
The dialog offers rebooting the system or just exiting setup. As setup leaves behind a `config.sys` with things like himem and smartdrv, it thinks a reboot makes sense. In fdpp that file is anyhow  ignored though. There is also an `autoexec.bat` which intends to set the path, but also this is ignored.

The only relevant thing after setup completes is adding `c:\windows` to the path I guess. This is handled by the current scripts.

Would it make sense to delete the unused `config.sys` and `autoexec.bat` that the installer leaves behind?

stsp commented:
> Would it make sense to delete the unused config.sys and autoexec.bat that the installer leaves behind?

Sounds good.
